cmake/thirdparty/BLTSetupHIP.cmake

```diff
@@ -87,14 +68,10 @@ else()
 endif()
 
 blt_import_library(NAME          blt_hip
-                   COMPILE_FLAGS ${_blt_hip_compile_flags}
                    EXPORTABLE    ${BLT_EXPORT_THIRDPARTY}
+                   DEPENDS_ON    blt_hip_runtime
```

Comment:
If this depends on blt_hip_runtime, shouldn't the target be created after the blt_hip_runtime target is created?

Reply:
Order of target creation doesn't matter.

Reply:
Good to know!